### DISCUSSION TITLE
Clickable components

### DISCUSSION THREAD

**Discussion by DevrayePatil** [Budibase/budibase#7076] [Help] [answered]
can we create clickable components in budibase. I have created Stat card, I have make it clickable. 
How can I achieve this?

**Comment by R2bEEaton** [comment, marked as answer]
Hey @DevrayePatil. Stat Cards currently do not have a click action (I think they should: #6244, #6788).

However, as shown in the linked discussions, regular cards do have click actions (see how to use them in #7080). You can use that instead of Stat Cards for now and use an Embed to make it look more like a Stat Card.

Input the following into an `Embed` element.
```
<style>
.spectrum-Card-footer {
    padding: 0px;
    color: var(--spectrum-link-primary-m-text-color);
    font-size: 2rem;
    font-weight: 600;
}
</style>
```

Now you can use the Title and Description of a regular Card and it should look mostly like a Stat Card. You can tweak the CSS to your liking if you want to do more with this.

**Reply by DevrayePatil** [reply]
can we filter data by clicking on this

**Reply by R2bEEaton** [reply]
You can definitely use the click action to Update State or a Field value (or something like that) which controls a Data Source filter. Or you can have it redirect to a different screen with a `:id` at the end and show something based on that on the other screen. Many ways to do this.

**Reply by DevrayePatil** [reply]
ok thank you
